### PR TITLE
iac: init at 0.1.7

### PR DESCRIPTION
**Pull request by hughobrien**
###### Description of changes

Introduce [iat](https://www.berlios.de/software/iso9660-analyzer-tool/) "Iso9660 Analyzer Tool", a simple utility that can convert from ISO format to IMG format.

There is a [github mirror](https://github.com/ropery/iat) available, but it appears to have no connection with the original project, so I have opted to use the sourceforge link, which they have endorsed.

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes (or backporting 23.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -889,6 +889,8 @@ with pkgs;
 
   graph-easy = callPackage ../tools/graphics/graph-easy { };
 
+  iat = callPackage ../development/tools/iat { };
```
Comment (ehmry) [inline pkgs/top-level/all-packages.nix:892]: Please find a more appropriate category. We will reorganize this hierarchy at some point but we still try to keep it sorted.

File: pkgs/development/tools/iat/default.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation (finalAttr: {
+  pname = "iat";
+  version = "0.1.7";
+
+  src = fetchurl {
+    url = "https://cfhcable.dl.sourceforge.net/project/iat.berlios/iat-0.1.7.tar.gz";
```
Comment (ehmry) [inline pkgs/development/tools/iat/default.nix:11]: Try and find a `mirror://sourceforge/` URL that would work here. There should be examples of doing this nearby.

**Review by ghost — CHANGES_REQUESTED** [review]

File: pkgs/top-level/all-packages.nix (hunk 1 above)
Comment (hughobrien) [reply pkgs/top-level/all-packages.nix:892]: Thank you, moved to `pkgs/tools/cd-dvd/`

File: pkgs/development/tools/iat/default.nix (hunk 2 above)
Comment (hughobrien) [reply pkgs/development/tools/iat/default.nix]: Nice catch, done.

**Pull request merged**